gum/gumdarwinmodule.c

```diff
@@ -30,6 +33,23 @@ typedef struct _GumEmitTermPointersContext GumEmitTermPointersContext;
 
 typedef struct _GumExportsTrieForeachContext GumExportsTrieForeachContext;
 
+typedef struct _GumChainedFixupsHeader GumChainedFixupsHeader;
+
+typedef struct _GumChainedStartsInImage GumChainedStartsInImage;
+typedef struct _GumChainedStartsInSegment GumChainedStartsInSegment;
+
+typedef uint32_t GumChainedImportFormat;
+typedef uint32_t GumChainedSymbolFormat;
```

Comment:
For consistency we should be using GLib types here, as this code is also compiled on Windows.